packages/react/src/primitives/Accordion/__tests__/Accordion.test.tsx

```diff
@@ -1,4 +1,5 @@
 import { render, screen } from '@testing-library/react';
+import { act } from '@testing-library/react-hooks';
```

Comment:
I don't think there are any hooks being tested in this file. Prefer to import act  from `'@testing-library/react'`